.gitignore

```diff
@@ -8,6 +8,9 @@ _targets
 !*/out/CONUS_1951_2020/
 */out/CONUS_1951_2020/*
 
+# Ignore images (can build locally with R and then put on S3)
```

Comment:
We need to either add this s3 step into the pipeline or commit the files as a temporary fix for now

Reply:
@aaarcher-usgs - I think as soon as we resolve this ^ we're good to go on this PR!